src/components/inputs/ColorSelect2/ColorSelect2.test.tsx

```diff
@@ -57,10 +57,7 @@ describe('ColorSelect2', () => {
     renderWithThemeProvider(<ColorSelect2 value={TEST_COLOR} />);
     const input = screen.getByLabelText('color');
     await act(async () => await userEvent.click(input));
-
-    const selectedColor = screen.getByLabelText('color preview');
-    const selectedColorValue = selectedColor.getAttribute('color');
-    expect(selectedColorValue).toBe(TEST_COLOR);
+    expect((input as HTMLInputElement).value).toBe(TEST_COLOR);
```

Comment:
Because it doesn't infer the type of input, return type of `getByLabelText` is `HTMLElement`
![Screenshot 2023-08-17 at 14 38 54](https://github.com/vimeo/iris/assets/59689809/ce387c68-0ed7-4826-8c4d-f1e6a0dfc178)